staticmap.php

```diff
@@ -28,16 +28,33 @@
 error_reporting(0);
 ini_set('display_errors', 'off');
 
-Class staticMapLite
+class staticMapLite
 {
 
     protected $maxWidth = 1024;
     protected $maxHeight = 1024;
 
     protected $tileSize = 256;
-    protected $tileSrcUrl = array('mapnik' => 'http://tile.openstreetmap.org/{Z}/{X}/{Y}.png',
-        'osmarenderer' => 'http://otile1.mqcdn.com/tiles/1.0.0/osm/{Z}/{X}/{Y}.png',
-        'cycle' => 'http://a.tile.opencyclemap.org/cycle/{Z}/{X}/{Y}.png',
+
+    protected $tileSrcUrl = array(
+        'mapnik'        => 'http://tile.openstreetmap.org/{Z}/{X}/{Y}.png',
+        'osm'               => 'http://tile.openstreetmap.de/tiles/osmde/{z}/{x}/{y}.png',
+        'osmarenderer'      => 'http://otile1.mqcdn.com/tiles/1.0.0/osm/{Z}/{X}/{Y}.png',
+        'cycle'             => 'http://a.tile.opencyclemap.org/cycle/{Z}/{X}/{Y}.png',
+        'stamen'            => 'http://a.tile.stamen.com/toner-hybrid/{Z}/{X}/{Y}.png',
```

Comment:
we should call that `stamen-toner` in case we want to add some of the other stamen styles